media/score.js

```diff
@@ -1,7 +1,8 @@
 // need to assign score dynamically
-const score = 18
-
+const score = localStorage.getItem('score')
+const amount = 20
```

Comment:
20 as in the amount of questions there  is? I think this could be dinamic too. Or if not, I think there are 50 questions in every trivia.